packages/template-cli/templates/src/ssr/components/ApiPane/index.tsx

```diff
@@ -12,11 +12,11 @@ type ApiPaneProps = {
 export const ApiPane = ({menuItems, isLoading}: ApiPaneProps) => {
     return (
         <Container item>
-            <Typography variant="h1">Latest menus:</Typography>
+            <Typography variant="h2">Latest menus:</Typography>
```

Comment:
This fails the aXe tests, you need at least one h1 on the page for accessibility. https://amido-dev.visualstudio.com/Amido-Stacks/_build/results?buildId=6304&view=logs&j=95fd2405-9ee1-5ffd-0e58-c04b67633907&t=07091dd7-b0ea-5595-026e-785d64cd745f